src/overlays/gamestates/ovl_title/z_title.c

```diff
@@ -8,11 +8,11 @@
 #include "alloca.h"
 #include "assets/textures/nintendo_rogo_static/nintendo_rogo_static.h"
 
-void ConsoleLogo_PrintBuildInfo(Gfx** gfxp) {
+void ConsoleLogo_PrintBuildInfo(Gfx** gfxP) {
     Gfx* g;
```

Comment:
```suggestion
    Gfx* gfx;
```
You don't have to if you don't want to, but since I saw it thought I would mention it.